src/SprykerEco/Shared/CrefoPayApi/CrefoPayApiConstants.php

```diff
@@ -17,6 +17,7 @@ interface CrefoPayApiConstants
      *  - Url for createTransaction API requests.
      *
      * @api
+     * @var string
```

Comment:
I checked this with @dereuromark there should be an empty line between @api and @var. For some reason the sniffer didn't detect it.

```suggestion
     * @api
     *
     * @var string
```

I reported it [here ](https://github.com/spryker/code-sniffer/issues/285)as discussed with @dereuromark.